<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditDialog</class>
 <widget class="QDialog" name="ItemEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="itemEdTabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="styleTab">
      <attribute name="title">
       <string>Item Style</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>281</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>Line</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>51</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Style  :   </string>
        </property>
       </widget>
       <widget class="QComboBox" name="pColorcomboBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>81</width>
          <height>27</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>black</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>red</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>51</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Color : </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>41</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Size : </string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>62</width>
          <height>27</height>
         </rect>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
       <widget class="QComboBox" name="lineComboBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>___________</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>_ _ _ _ _ _ _ _ </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>. . . . . . . . . . . .</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>_ . _ . _ . _ . _ . </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>_ . . _ . . _ . . _</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>281</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>Area</string>
       </property>
       <widget class="QPushButton" name="customButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>30</y>
          <width>61</width>
          <height>27</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Custom</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>51</width>
          <height>27</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Color :</string>
        </property>
       </widget>
       <widget class="QComboBox" name="bColorcomboBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>white</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>black</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>yellow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>orange</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pink</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Other...</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>51</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Style  :   </string>
        </property>
       </widget>
       <widget class="QComboBox" name="brushComboBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtBottom</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>16</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>No Brush</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SolidPattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense1Pattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense2Pattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense3Pattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense4Pattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense5Pattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense6Pattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dense7Pattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HorPattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VerPattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CrossPattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BDiagPattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FDiagPattern</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LinearGradient</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RadialGradient</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ConicalGradient</string>
         </property>
        </item>
       </widget>
       <zorder>customButton</zorder>
       <zorder>label</zorder>
       <zorder>bColorcomboBox</zorder>
       <zorder>label_2</zorder>
       <zorder>brushComboBox</zorder>
       <zorder>label_5</zorder>
       <zorder>brushComboBox</zorder>
      </widget>
     </widget>
     <widget class="QWidget" name="randrTab">
      <attribute name="title">
       <string>Rotation and Resize</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>291</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Resize</string>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>117</width>
          <height>27</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Width : </string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>117</width>
          <height>27</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Height :</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_4"/>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>301</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Rotate</string>
       </property>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>121</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>Rotate from :</string>
        </property>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>116</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Center</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>116</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Top Left</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>116</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Top Right</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>80</y>
           <width>116</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Bottom Left</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>100</y>
           <width>116</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Buttom Right</string>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>121</width>
          <height>27</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Digree : </string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox"/>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customButton</sender>
   <signal>clicked()</signal>
   <receiver>ItemEditDialog</receiver>
   <slot>customColorAct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>customColorAct()</slot>
 </slots>
</ui>
